src/stores/useDashboard.ts

```
@@ -179,13 +182,13 @@ export function fromLocal(lc: LocalConfig): ChainConfig {
-  if(lc.provider_chain) {
+  if (lc.provider_chain) {
     conf.providerChain = {
-      api: apiConverter(lc.provider_chain.api)
-    }
+      api: apiConverter(lc.provider_chain.api),
+    };
   }
-  conf.features = lc.features
-  conf.logo = lc.logo.startsWith('http') ? lc.logo : `https://ping.pub${lc.logo}`;
+  conf.features = lc.features;
+  conf.logo = lc.logo && lc.logo.startsWith('http') ? lc.logo : `${lc.logo}`;
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)